src/helpers/helpers.core.js

```diff
@@ -306,6 +306,18 @@ var helpers = {
 
 		ChartElement.__super__ = me.prototype;
 		return ChartElement;
+	},
+
+	/**
+	 * If argument is string, return it. If it is an array, join its elements using
+	 * a space string (' ').
+	 * @private
+	 */
+	_joinIfArray: function(stringOrArray) {
+		if (Array.isArray(stringOrArray)) {
```

Comment:
should use `helpers.isArray` here in place of `Array.isArray`

Reply:
![you](https://media.giphy.com/media/riQ6xcameX84w/giphy.gif)

Done :)